stix_shifter_modules/cisco_secure_email/stix_transmission/error_mapper.py

```diff
@@ -0,0 +1,39 @@
+from stix_shifter_utils.utils.error_mapper_base import ErrorMapperBase
+from stix_shifter_utils.utils.error_response import ErrorCode
+from stix_shifter_utils.utils import logger
+
+error_mapping = {
+    100: ErrorCode.TRANSMISSION_INVALID_PARAMETER,
+    406: ErrorCode.TRANSMISSION_CONNECT,
+    408: ErrorCode.TRANSMISSION_CONNECT,
+    503: ErrorCode.TRANSMISSION_CONNECT,
+    400: ErrorCode.TRANSMISSION_QUERY_PARSING_ERROR,
+    401: ErrorCode.TRANSMISSION_AUTH_CREDENTIALS,
+    403: ErrorCode.TRANSMISSION_FORBIDDEN,
+    404: ErrorCode.TRANSMISSION_SEARCH_DOES_NOT_EXISTS,
```

Comment:
i think this should be invalid query. `no_results` error code is not appropriate. this error code usually returns when query syntax is invalid

Reply:
Updated error code as invalid_query